tests/automation_framework/env.py

```diff
@@ -44,3 +44,8 @@
     result_path = os.path.join(currentPath, "results")
     if not os.path.exists(result_path):
         os.mkdir(result_path)
+
```

Comment:
Moved the if proxy mode condition from all worker test PY files to env.py. Forgot to run pep_8, hence the end of file error, will fix in the next PR.